src/test/scala/wdl4s/SyntaxErrorSpec.scala

```diff
@@ -945,6 +947,44 @@ class SyntaxErrorSpec extends FlatSpec with Matchers {
       """.stripMargin
   }
 
+  case object InvalidInputMapping extends ErrorWdl {
```

Comment:
Could we also have a test for the new `undefinedIdentified` formatter? Or even also include an undefined identifier in this one and check that we get both error messages?